airdrop/infrastructure/repositories/user_repository.py

```diff
@@ -26,14 +25,32 @@ def subscribe_to_notifications(self, email):
             raise e
 
     def check_rewards_awarded(self, airdrop_id, airdrop_window_id, address):
+
+        try:
+            is_eligible = (
```

Comment:
We need to add two conditions
- User should be in registration and rewards table
- Rewards should be greater than 0
We can have user records with 0 rewards

Reply:
Eligibilty is determined by an entry in the user_rewards table for a given window ID and agree the rewards should be >  0 
However you cannot chekc for an entryin the user_registration table for eligiility ( consider non ocam scenarios also) 
Users can register if they are eligible